### ISSUE TITLE
Horizontal Button Stack overlaps main dashboard controls with larger screen sizes 

### ISSUE THREAD

**Issue by lennon101** [Clooos/Bubble-Card#818]
**Describe the bug**  

**Working state:** 
I havn't figured out the exact screen size where this occurs, but if the screen size is small (on mobile or smaller desktop screens) the horizontal button stack is displayed correctly and does not overlap the other contros (as shown below). 

![2024-09-23_09 03 34_ShareX Overview_–_Home_Assistant_-_Google_Chrome](https://github.com/user-attachments/assets/877691fc-ef5e-4068-af0e-fbfb94581373)

**Not working state**
Below is a screenshot of a larger screen and and you can see how the cards on the main dashboard are now hidden by the horizontal stack and I cannot scroll down any further to reveal them. This behaviour only occurs in larger screen sizes. 

![2024-09-23_09 04 57_ShareX](https://github.com/user-attachments/assets/166f1826-fd9b-4162-a6bd-b055abfed36f)

**To Reproduce**  
1. Add a Horizontal Button Stack with some buttons and place it on a dashboard with other cards 
2. Change the screen size (by decreasing the window size of chrome or other web browser) and observe how the Horizontal Button Stack does not hide the other cards at the very bottom of the dashboard 
3. Now increase the screen size to desktop size or larger and observer how the Horizontal Button Stack now hides the cards on the main dashboard (the cards at the bottom). 

**Expected behavior**  
Horizontal Button Stack cards should not hide main dashboard cards

**Screenshots**  
See above. 

**YAML**  

Not  sure if relevant, but here is my yaml for the Horizontal Button Stack shown in the screenshots 

```yaml
type: custom:bubble-card
card_type: horizontal-buttons-stack
auto_order: false
1_name: Lights
1_icon: mdi:lightbulb-group
1_link: '#all-lights'
2_name: Fans
2_icon: mdi:fan
2_link: '#all-fans'
2_entity: fan.all_fans
1_entity: light.all_lights
```

**Informations (please complete the following information):**  
 - OS: behaviour has been observed in chrome on windows and the IOS HA app
 - Browser/App: [e.g. chrome, safari]
 - Bubble Card version: v2.2.4
 - Home Assistant version: core 2024.9.1

**Additional context**  
I don't think I have any further infromation to add at this stage. Would just like to add my appreciation to this amazing integration! You've done an incredible job and I reckon I owe you a coffee! ☕

Thank you! 🍻

**Comment by lennon101** [comment]
Update: It seems related to the width of the browser/screen instead of the actual size of the screen itself. I hope that makes sense.

**Comment by t-flow-sash** [comment]
Giving this a +1.
I am experiencing the same issue on the HA App on iOS with an iPhone 14 Pro Max as well as on my Wall Unit which is a Lenovo Tab M10 using Fully Kiosk Browser.

The Bubble Card does not allow me to fully scroll down leaving out a last line of elements.

**Comment by Clooos** [comment]
Hi and thank you for the feedback!

Just to be sure, have you added your HBS as the last card of your dashboard? If yes this is indeed an issue, maybe introduced in one of the new HA releases.

And thank you for the coffee by the way! 😀

**Comment by t-flow-sash** [comment]
Right! The buttons are the last pieces of the bubble card at the very bottom. 
I was promoting your bubble card in one of my recent "Best Dashboard" tutorial videos on YouTube and a user was asking me if I was experiencing the same. Didn't notice so far but now the issue occurs.

Nevermind. You're doing a great job here and Bubble Card definitely is one of the masterpiece Lovelace Integrations you can get out there! Thanks for doing all this and looking forward to a fix some time in the future :)

**Comment by Clooos** [comment]
Thank you so much for the kind words! I really need to check that video! 😀

**Comment by Clooos** [comment]
You can share it here, and indeed the auto translation should be enough to understand it 😁

**Comment by t-flow-sash** [comment]
Here you go 😀 

https://youtu.be/oZzGZGn8ycA?si=JvlrWO0PUR0gqfL_

It's part 3 of a playlist creating a beautiful dashboard 😁

**Comment by Clooos** [comment]
Thank you for this! In fact, I've already watched this one before by doing my monthly Bubble Card YouTube research 🤓

To get back to our topic, I will take a look at this issue 😁

**Comment by t-flow-sash** [comment]
Awesome! Hope you enjoyed! 

And thanks so much for looking into it!
